<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>system.ico</normaloff>system.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
background-color: rgb(0, 85, 127);
}

QGroupBox{
font: 20pt &quot;宋体&quot;;
color: rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:8px;
margin-top:22px;
}
QGroupBox:title{
subcontrol-origin: margin;
top:10px;
left: 10px;
}</string>
   </property>
   <widget class="QLabel" name="lbBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;宋体&quot;;
color:rgb(0, 122, 165);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>电压/电流探测器检测工装</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1024</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: rgb(0, 85, 127);
}
QToolButton{
font: 28pt &quot;宋体&quot;;
color:  rgb(0, 85, 127);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <widget class="QWidget" name="pageMain">
     <widget class="QToolButton" name="tBtnLeak">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>200</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>漏电</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Image/login.png</normaloff>:/Image/login.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnThreeVol">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>180</y>
        <width>200</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>3V/2V/V</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Image/tools.png</normaloff>:/Image/tools.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnThreeVolCur">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>200</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>2VA/VA</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Image/selfCheck.png</normaloff>:/Image/selfCheck.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnQuit">
      <property name="geometry">
       <rect>
        <x>885</x>
        <y>460</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Image/tools.png</normaloff>:/Image/tools.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnSingleVolCur">
      <property name="geometry">
       <rect>
        <x>765</x>
        <y>180</y>
        <width>200</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>6V3A</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Image/selfCheck.png</normaloff>:/Image/selfCheck.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLeak"/>
    <widget class="QWidget" name="pageThreeCurVol"/>
    <widget class="QWidget" name="pageThreeVol"/>
    <widget class="QWidget" name="pageSingelCurVol"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
